models/classes/accessControl/ContextInterface.php

```diff
@@ -22,8 +22,9 @@
 
 namespace oat\tao\model\accessControl;
 
-use oat\oatbox\user\User;
-
+/**
+ * @deprecated Use \oat\tao\model\Context\ContextInterface
+ */
 interface ContextInterface
```

Comment:
You can make this interface extends the proper one and to not keep the same methods in 2 places.

```suggestion
interface ContextInterface extends \oat\tao\model\Context\ContextInterface
```